.husky/pre-push

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 cd app
-pnpm run test:unit
+docker compose up -d tests
```

Comment:
Can we replace this 2 lines with a new make command?

I think it makes sense given that we have all those make commands.

I would suggest to add a new one called `make tests-nowatch` for this 2 lines.
The reason is that the one that will be usually used when developing will be `make tests` which is more comfortable to keep it like that, and have the one without watching being a longer syntax and not the other way around.

What do you think?

PS: Update the make help command too.

Reply:
Yes, I did consider that option as well, but I wasn't certain if it was the right decision. I thought that command would likely only be used in this particular scenario, so I didn't see the necessity of adding a new "make" command. However, I  understand it's part of our workflow with Docker, so it makes sense to include it.